### PR TITLE
feat(CORSO-45): View All Services

### PR DESCRIPTION
**Pull request by KarinaSofia**
JIRA: https://champlainsaintlambert.atlassian.net/browse/CORSO-45?atlOrigin=eyJpIjoiNWUwZDQ2ZDZmYmQ3NDY1MGJlNDRiMTk4NmQzYmNhMmUiLCJwIjoiaiJ9
## Context:
Implementing the View All Services Feature
## Changes
Added the view all service feature and created tests for that feature
## Before and After UI 
Before (component not implemented)
image
After
image

### CODE REVIEW

File: corso_website_fe/src/Main Page Components/AllServices.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+            <div className="row">
+                {services.map(service => (
+                    <div className="col-lg-4 col-md-6 col-sm-12 mb-4" key={service.serviceId}>
+                        <div className="service-card">
+                            <div className="service-icon-container">
+                                <img src={service.serviceIcon} alt={`${service.serviceName} icon`} className="service-icon" />
+                            </div>
+                            <div className="service-info">
+                                <h2 className="service-name">{service.serviceName}</h2>
+                                {/* <a className="details-hyperlink" href="#">View Details</a> */}
```
Comment (Mila5847) [inline corso_website_fe/src/Main Page Components/AllServices.tsx:44]: Did you mean to leave this line commented?

**Review by Mila5847 — APPROVED** [review]
Services looks really great, good job! The code is also tested (back-end and E2E), nice!

**Review by DylanBrass — APPROVED** [review]
All good, not much changed but what did looks good !

File: corso_website_springboot/src/main/java/com/corso/springboot/Service_Subdomain/presentationlayer/ServiceController.java [hunk 2]
```diff
@@ -23,4 +23,9 @@ public ServiceController(ServiceService serviceService) {
     public ResponseEntity<ServiceResponse> getServiceById(@PathVariable String serviceId) {
         return ResponseEntity.status(HttpStatus.OK).body(serviceService.getServiceByServiceId(serviceId));
     }
+
+    @GetMapping("/allServices")
+    public ResponseEntity<List<ServiceResponse>> getAllServices(){
+        return ResponseEntity.status(HttpStatus.OK).body(serviceService.getAllServices());
+    }
```
Comment (DylanBrass) [inline corso_website_springboot/src/main/java/com/corso/springboot/Service_Subdomain/presentationlayer/ServiceController.java:30]: Where is the get by Id ?

**Review by DylanBrass — APPROVED** [review]

File: corso_website_fe/src/Main Page Components/AllServices.tsx (hunk 1 above)
Comment (KarinaSofia) [reply corso_website_fe/src/Main Page Components/AllServices.tsx:44]: Yes, it will be used for my next story!

**Pull request merged**